tests/e2e/test_rosacli_node_pool.go

```diff
@@ -1713,4 +1714,53 @@ var _ = Describe("Edit nodepool",
 				ContainSubstring(`Max replicas`))
 			Expect(mpDescription.String()).Should(MatchRegexp(`Desired replicas:[\s\t]*0`))
 		})
+
+		It("create/describe rosa hcp machinepool support imdsv2 - [id:75227]",
+			labels.Critical, labels.Runtime.Day2,
+			func() {
+				By("Check the help message of 'rosa create machinepool -h'")
+				res, err := machinePoolService.RetrieveHelpForCreate()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(res.String()).To(ContainSubstring("--ec2-metadata-http-tokens"))
+
+				imdsv2Values := []string{"",
+					constants.OptionalEc2MetadataHttpTokens,
+					constants.RequiredEc2MetadataHttpTokens}
+
+				for _, imdsv2Value := range imdsv2Values {
+					machinePool_Name := common.GenerateRandomName("ocp-75227", 2)
+					By("Create a machinepool to the cluster by setting different values of --ec2-metadata-http-tokens")
```

Comment:
I'd like the Description like this `By("Create a machinepool with --ec2-metadata-http-tokens = " + imdsv2Value)`

Reply:
Okay will change it, Thanks!

Reply:
@yasun1 Changed it, PTAL , Thanks!